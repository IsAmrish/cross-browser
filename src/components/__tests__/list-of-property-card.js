import React from "react";
import renderer from "react-test-renderer";
import ListOfPropertyCard from "../list-of-property-card";

describe('ListOfPropertyCard component test', () => {
  it('Render ListOfPropertyCard', () => {
    const tree = renderer.create(<ListOfPropertyCard />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});