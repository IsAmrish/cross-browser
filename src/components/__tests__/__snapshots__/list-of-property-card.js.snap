// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ListOfPropertyCard component test Render ListOfPropertyCard 1`] = `
Array [
  <div
    style={
      Object {
        "marginBottom": "20px",
      }
    }
  >
    <div
      style={
        Object {
          "border": "1px solid grey",
          "padding": "30px",
        }
      }
    >
      <div
        style={
          Object {
            "background": "lightblue",
            "marginBottom": "30px",
            "padding": "16px",
          }
        }
      >
        <h2
          style={
            Object {
              "margin": "0px",
            }
          }
        >
          backface-visibility property
        </h2>
      </div>
      <div
        style={
          Object {
            "backgroundColor": "#c2ddf0",
            "padding": "16px",
          }
        }
      >
        <h3>
          How to use it
        </h3>
        <code>
          <pre
            style={
              Object {
                "backgroundColor": "#e4ecf2",
              }
            }
          >
            
selector {
  -webkit-backface-visibility: hidden; /* Safari */
  backface-visibility: hidden;
}

          </pre>
        </code>
      </div>
    </div>
  </div>,
  <div
    style={
      Object {
        "marginBottom": "20px",
      }
    }
  >
    <div
      style={
        Object {
          "border": "1px solid grey",
          "padding": "30px",
        }
      }
    >
      <div
        style={
          Object {
            "background": "lightblue",
            "marginBottom": "30px",
            "padding": "16px",
          }
        }
      >
        <h2
          style={
            Object {
              "margin": "0px",
            }
          }
        >
          hyphens property
        </h2>
      </div>
      <div
        style={
          Object {
            "backgroundColor": "#c2ddf0",
            "padding": "16px",
          }
        }
      >
        <h3>
          How to use it
        </h3>
        <code>
          <pre
            style={
              Object {
                "backgroundColor": "#e4ecf2",
              }
            }
          >
            
selector {
  -webkit-hyphens: auto; 
  -ms-hyphens: auto;
  hyphens: auto;
}

          </pre>
        </code>
      </div>
    </div>
  </div>,
  <div
    style={
      Object {
        "marginBottom": "20px",
      }
    }
  >
    <div
      style={
        Object {
          "border": "1px solid grey",
          "padding": "30px",
        }
      }
    >
      <div
        style={
          Object {
            "background": "lightblue",
            "marginBottom": "30px",
            "padding": "16px",
          }
        }
      >
        <h2
          style={
            Object {
              "margin": "0px",
            }
          }
        >
          user-select property
        </h2>
      </div>
      <div
        style={
          Object {
            "backgroundColor": "#c2ddf0",
            "padding": "16px",
          }
        }
      >
        <h3>
          How to use it
        </h3>
        <code>
          <pre
            style={
              Object {
                "backgroundColor": "#e4ecf2",
              }
            }
          >
            
selector {
  -webkit-user-select: none; 
  -ms-user-select: none;
  user-select: none;
}

          </pre>
        </code>
      </div>
    </div>
  </div>,
]
`;
